.github/workflows/ai-runner-pipelines-fps-test.yaml

```diff
@@ -0,0 +1,87 @@
+name: FPS check for live-video-to-video pipelines
+
+on:
+  pull_request:
+    paths:
+      - "runner/**"
+      - "!runner/.devcontainer/**"
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+    paths:
+      - "runner/**"
+      - "!runner/.devcontainer/**"
+  workflow_dispatch:
+
+jobs:
+  run-fps-test:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        model_config:
+          - { id: noop, input_fps: 30, exp_output_fps: 28 }
+          - { id: liveportrait, input_fps: 30, exp_output_fps: 28 }
+          - { id: streamdiffusion, input_fps: 30, exp_output_fps: 25 }
+          - { id: comfyui, input_fps: 30, exp_output_fps: 21 }
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.23.3'
+
+      - name: Build Docker images
+        env:
+          MODEL_ID: ${{ matrix.model_config.id }}
+        run: |
+          cd runner
+          docker build -t livepeer/ai-runner:live-base -f docker/Dockerfile.live-base .
+          if [ "${MODEL_ID}" = "noop" ]; then
+              docker build -t livepeer/ai-runner:live-app-noop -f docker/Dockerfile.live-app-noop .
+          else
+              docker build -t livepeer/ai-runner:live-base-${MODEL_ID} -f docker/Dockerfile.live-base-${MODEL_ID} .
+              docker build -t livepeer/ai-runner:live-app-${MODEL_ID} -f docker/Dockerfile.live-app__PIPELINE__ --build-arg PIPELINE=${MODEL_ID} .
+          fi
+          cd ..
```

Comment:
- if we use the same node each time, which is the case, the layers are cached anyway right? but yeah if docker build workflow already exists then can just try to use that
- i've added base because we have to account for changes in req packages -- maybe can only run base docker build only when dockerfiles or req files are updated
- i think its important that there is caching, otherwise any optimizations won't cut it

Reply:
with the self-hosted runners, we clean up disk frequently, so that we dont run out of disk space for other jobs.

this means the intermediate layers etc. are also gone. as victor mentioned above (and i mentioned in the discord thread), best would be to trigger this workflow after docker build has finished and pull those images for testing.

Reply:
@hjpotter92  can we not wipe the models dir if possible? that would save a lot of time
@victorges  thoughts?